rlberry/manager/evaluation.py

```diff
@@ -96,19 +95,19 @@ def evaluate_agents(
     return output
 
 
-def read_writer_data(agent_manager, tag, preprocess_func=None):
+def read_writer_data(data_source, tag, preprocess_func=None):
     """
-    Given a list of AgentManager, read data (corresponding to info) obtained in each episode.
+    Given a list of AgentManager or a folder, read data (corresponding to info) obtained in each episode.
     The dictionary returned by agents' .fit() method must contain a key equal to `info`.
 
-    This function can be called with agents that have been trained beforhand and
-    which have been saved in `agent_manager.output_dir`. The last session for each
-    individual agent will then  be loaded from the pickle files in `agent_manager.output_dir`.
-
     Parameters
     ----------
-    agent_manager : AgentManager, or list of AgentManager or directory
-        If AgentManager or list of AgentManager, load data from `agent_manager.output_dir`.
+    data_source : AgentManager, or list of AgentManager or directory or list of directories
```

Comment:
When it can be a directory (or a list), what kind of directory are we expecting? Are those directories that we have as `output_dir` to `AgentManager`? Are they strings or Path objects? I think we should add this info to the documentation.